### DISCUSSION TITLE
Using npm as a vessel

### DISCUSSION THREAD

**Discussion by mrmetaverse** [mrmetaverse/soiree#8] [Q&A] [answered]
What are the pro's and con's of using NPM as our delivery mechanism?

**Comment by fire** [comment, marked as answer]
Un-sorted pros and cons.

* Javascript is a extremely popular language
* A path for WASM is useful for modding
* Javascript is not a native library
* Unlike Godot Engine, NPM is not a mono repo
* Electron is good
* Electron is bad
* Requires online access I think. Might be an offline archive of npm.
* Non-native code
* JS JIT may be faster than average c code, but not faster than the best c/c++ code.
